src/python/README.md

```diff
@@ -4,10 +4,10 @@ Simple client for passing relevant events from MTG Arena logs to the 17Lands RES
 
 ## Usage
 
-To run, simply enter `seventeelands` in your terminal. On first run, you will be prompted for your user token.
+To run, simply enter `seventeenlands` in your terminal. On first run, you will be prompted for your user token.
 
 ## Notes
 
 Licensed under GNU GPL v3.0 (see included LICENSE).
 
-This MTGA log follower is unofficial Fan Content permitted under the Fan Content Policy. Not approved/endorsed by Wizards. Portions of the materials used are property of Wizards of the Coast. ©Wizards of the Coast LLC. See https://company.wizards.com/fancontentpolicy for more details.
```

Comment:
I don't know why there's a highlighted line here, I edited it directly on GitHub. Maybe something to do with line endings?

Reply:
Ah yeah, that's probably it